{
  "contest": 375,
  "handle": "pwning_dev",
  "risk": "3",
  "title": "'withdraw()' function of the VulnerableContract is susceptible to reentrancy attacks due to the order of operations",
  "issueId": 82,
  "issueUrl": "https://github.com/code-423n4/2024-05-arbitrum-foundation-validation/issues/82"
}